tests/test_diffraction_StaticStructureFactorDirect.py

```diff
@@ -0,0 +1,248 @@
+import matplotlib
```

Comment:
Could this be resolved directly by constructing the base class (see below) and then testing the base class?

Reply:
I would not advise this. The base class should stay abstract since it doesn't really represent a method of calculating the structure factor, it should just exist to prevent defining the same properties and documenting them multiple times.